abb_hardware_interface/package.xml

```diff
@@ -21,9 +21,9 @@
   <depend>rclcpp_lifecycle</depend>
 
   <test_depend>ament_cmake_gtest</test_depend>
-
   <test_depend>ament_lint_auto</test_depend>
   <test_depend>ament_lint_cmake</test_depend>
+  <test_depend>ros_testing</test_depend>
```

Comment:
I'm not sure the `ros_testing` dep is needed here since no tests run for this package. Am I missing something?

Reply:
Nope, I originally had the test in the hardware interface package, but I moved it out and must have missed this, thanks for the catch!